### ISSUE TITLE
Define development work-flow

### ISSUE THREAD

**Issue by nikaburu** [fullstackjsIE/momentum#1]
Hey all, 

the one of the fundamental question that we need to solve first is a development workflow strategy for this project.   

I've found this article clear to see pros and cons of different strategies: [Comparing Workflows](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)

Let's discuss it here. Any thoughts?

**Comment by dpcrespo** [comment]
Hi nikaburu,

It seems pretty good, at my work we use the first flow. but I think both of them could be work. Personally I prefer the first one.

**Comment by lecoeurseb** [comment]
Yes we definitely need to talk about the Git workflow but could we start to talk about the features that we want to see on our web page, taking example of the links I gave you last time?
https://momentumdash.com/
https://destinations.momentumdash.com/
https://chrome.google.com/webstore/detail/momentum/laookkfknpbbblfpciffpaejjkokdgca

**Comment by nikaburu** [comment]
Hey, 

@dpcrespo, I agree that the centralised approach may be the best to start here. I'm not too experienced in others.

@lecoeurseb you are 100% right with documenting the requirements first. Let's rise it on Sunday, we may to show how to collaborate on markdown documents on github - will be a good exercise
